examples/AN00221_freq_domain_example/src/app_freq_domain_example.xc

```diff
@@ -68,6 +69,23 @@ multichannel_audio_block_s triple_buffer[3];
 //Data memory for the lib_mic_array decimation FIRs
 int data[8][THIRD_STAGE_COEFS_PER_STAGE*DECIMATION_FACTOR];
 
+void apply_window_function(lib_dsp_fft_complex_t p[], int window[]) {
```

Comment:
This should take 'unsigned N' as a parameter so when the length is changed it still works